api/tests/test_game_state_server.py

```diff
@@ -179,18 +230,20 @@ async def test_delete_non_existent_token(gss_with_client):
             TEST_REQUEST_ID,
         )
     )
-    assert reply[0].contents.type == 'error'
+    assert_all_match(
+        reply, [{'contents': {'type': 'error'}}, {'contents': {'type': 'state'}}]
```

Comment:
I think you've struck the right balance here for validating only the properties that are relevant for the test 👍

Reply:
Are you suggesting I slim down the other tests to match, or just commenting in general?

Reply:
No, I think you did a good job in this whole file